manifests/init.pp

```diff
@@ -24,7 +24,7 @@
 #                                           Can also install only server packages with value
 #                                           of 'server' or only agent packages with 'agent'.
 #
-# $port::                                   Override the port of the master we connect to.
+# $port::                                   Override the port of the server we connect to.
```

Comment:
I've thought about this and wondered if we need to rename it to `agent_server_port`, similar to how you changed `puppetmaster` to the more descriptive `agent_server_hostname`.